pkg/controller/feed/reconcile_test.go

```diff
@@ -369,6 +395,17 @@ func getStartInProgressFeed() *feedsv1alpha1.Feed {
 	return feed
 }
 
+func getSecretStartInProgressFeed() *feedsv1alpha1.Feed {
```

Comment:
@dprotaso The builder code from https://github.com/knative/serving/pull/1762 would have been helpful here.